archspec/cli.py

```diff
@@ -13,6 +13,7 @@
 
 
 @click.group(name="archspec")
+@click.version_option()
```

Comment:
Reading [here](https://click.palletsprojects.com/en/7.x/api/#click.version_option). I think we could do:
```suggestion
@click.version_option(version=archspec.__version__)
```
to avoid relying on any external logic for something as simple as printing the version.